index.html

```diff
@@ -29,6 +29,13 @@
     <header class="header">
```

Comment:
- Hey, great job with your design, however, there's a part of the design you've added in the mobile version which is not part of the Figma design as this was seen as the Figma design was been taken as a screenshot from a mobile device. Please, kindly remove this part as seen in the image below:

![image](https://github.com/henrycadenaec/PortfolioFinishMobileVersion/assets/59397159/27c3a1b4-0be0-438e-9b8e-7cbd665f91e5)